spec/views/hyrax/admin/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe 'hyrax/admin/users/index.html.erb', type: :view do
+  let(:presenter) { Hyrax::Admin::UsersPresenter.new }
+  let(:users) { [] }
+
+  before do
+    (1..11).each { |i| users << FactoryGirl.create(:user, display_name: "user#{i}", email: "email#{i}@example.com") }
```

Comment:
instead of creating these users, can we `build`them instead?  Then it should go faster as there's no persistence.